package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "stylelint-config-skyscanner",
-  "version": "2.0.0-beta.2",
+  "version": "2.0.0-beta.3",
```

Comment:
Don't update this yourself - `npx np` does this for us.

Reply:
Will revert that change 👍